mllib/src/test/scala/org/apache/spark/mllib/classification/LogisticRegressionSuite.scala

```diff
@@ -459,7 +461,41 @@ class LogisticRegressionSuite extends FunSuite with MLlibTestSparkContext with M
     // very steep curve in logistic function so that when we draw samples from distribution, it's
     // very easy to assign to another labels. However, this prediction result is consistent to R.
     validatePrediction(model.predict(validationRDD.map(_.features)).collect(), validationData, 0.47)
+  }
+
+  test("model export/import") {
```

Comment:
I have a question about upgradability. When we have `V2`, how to test `ImporterV1`?